reference-stores/next/pages/products/[handle].js

```diff
@@ -48,11 +48,11 @@ export async function getStaticProps({ params: { handle } }) {
   });
   const { page } = await resolvePageData({
     client: nacelleClient,
-    page: pages.edges[0]?.node
+    page: data.pages.edges[0]?.node
```

Comment:
Might be a good opportunity to use optional chaining with `.at`:


```suggestion
    page: data.pages.edges.at(0)?.node
```